Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_locate.cpp

```diff
@@ -201,22 +213,31 @@ void test_constructions(const G& g,
                                                        .geom_traits(K())));
 
     const FT sq_dist = CGAL::squared_distance(barycentric_pt, barycentric_pt_2);
-    assert(is_equal(sq_dist, FT(0)));
+    if (std::is_same<K, EPECK>()) {
+      assert(is_equal(sq_dist, FT(0)));
+    }
   }
 
   // ---------------------------------------------------------------------------
   loc = std::make_pair(f, CGAL::make_array(FT(0.3), FT(0.4), FT(0.3)));
   descriptor_variant dv = PMP::get_descriptor_from_location(loc, g);
   const face_descriptor* fd = boost::get<face_descriptor>(&dv);
-  assert(fd);
+  if (std::is_same<K, EPECK>()) {
+    assert(fd);
+  }
 
   loc = std::make_pair(f, CGAL::make_array(FT(0.5), FT(0.5), FT(0)));
   dv = PMP::get_descriptor_from_location(loc, g);
   const halfedge_descriptor* hd = boost::get<halfedge_descriptor>(&dv);
-  assert(hd);
+  if (std::is_same<K, EPECK>()) {
+    assert(hd);
+  }
 
   loc = std::make_pair(f, CGAL::make_array(FT(1), FT(0), FT(0)));
-  assert(PMP::is_on_vertex(loc, source(halfedge(f, g), g), g));
+  if (std::is_same<K, EPECK>()) {
+    assert(PMP::is_on_vertex(loc, source(halfedge(f, g), g), g));
+  }
+
```

Comment:
These are not necessary (these checks are exact inputs and combinatorial checks)